src/main/java/base/navigationservice/KeyboardListener.java

```diff
@@ -26,6 +26,9 @@ public void keyPressed(KeyEvent event) {
         if (keys[KeyEvent.VK_CONTROL] && keys[KeyEvent.VK_S]) {
             game.handleCTRLandS();
         }
+        if(keys[KeyEvent.VK_M]){ //press M
```

Comment:
Hi. Thank you for participating in my project :)
I have a few comments. 
1. Let's assign another key, I think H suits here better (as H for Home), since this function will teleport player to Home map. And M will be reserved to show minimap later